Packs/PaloAltoNetworksAutomaticSLR_Community/Integrations/PaloAltoNetworksAutomaticSLRCommunity/PaloAltoNetworksAutomaticSLRCommunity.py

```diff
@@ -508,7 +509,7 @@ def main():
     try:
 
         if ngfw_tls_verify is False or csp_tls_verify is False:
-            requests.packages.urllib3.disable_warnings()
+            urllib3.disable_warnings()
```

Comment:
since `disable_warnings` is called in CSP, I think we can remove this part

Reply:
Because I saw it was under `if`, I didn't want to get into it

Reply:
@dorschw Do you think the whole condition should be removed?